tray.c

```diff
@@ -71,6 +72,27 @@ CreateBitmaps()
     }
 }
 
+/*
+ * Reallocate hMenuConn as required. In case of not enough memory,
+ * o.num_config is reset to max space available value so that the
+ * program can continue.
+ */
+void
+allocate_connection_menu()
```

Comment:
All other functions in this file use CamelCase.

Reply:
Thanks for the review. Will change the function name for consistency.